sklearn/utils/extmath.py

```diff
@@ -374,6 +374,126 @@ def randomized_svd(M, n_components, *, n_oversamples=10, n_iter='auto',
         return U[:, :n_components], s[:n_components], Vt[:n_components, :]
 
 
+@_deprecate_positional_args
+def _randomized_eigsh(M, n_components, *, n_oversamples=10, n_iter='auto',
+                      power_iteration_normalizer='auto',
+                      selection='module', random_state=0):
```

Comment:
We are moving away from seeding with 0. Please use `None` as the default instead. See: #19670 for the context.

Reply:
ok. done !